gradle/release.gradle

```diff
@@ -1,30 +1,21 @@
+/**
+ * This Plugin is used to:
+ *  - create release artifacts of Sparkling Water
+ *  - create tag on Github pointing to the release hash
+ *  - create commit with new version on Github
+ */
 apply plugin: 'net.researchgate.release'
-apply plugin: 'nexus-workflow'
 
 release {
-  tagTemplate = 'RELEASE-$version'
-  // Safe point - do releases only from release branch - can be deleted in future
-  failOnUnversionedFiles = false
-  failOnCommitNeeded = false
-  preCommitText = ":tada: "
-  git {
-    // Note: the releases are only allowed from master branch
-    // requireBranch = '<<FILL ME>>'
-    // DEBUG: disable dry run in production/CI
-    // pushOptions = ["--dry-run"]
-  }
+    tagTemplate = 'RELEASE-$version'
+    // Safe point - do releases only from release branch - can be deleted in future
+    failOnUnversionedFiles = false
+    failOnCommitNeeded = false
+    preCommitText = ":tada: "
+    git {
+        // Note: the releases are only allowed from master branch
+        requireBranch = '<<FILL ME>>'
+        // DEBUG: disable dry run in production/CI
+        // pushOptions = ["--dry-run"]
+    }
```

Comment:
Separate the stages as there was some cradle bug not probably launching all `publish` tasks

Reply:
? follow documentation of publish plugin - it needs to be invoked per subproject

Reply:
This PR didn’t go in, but publishing and releasing can be separated. Currently we get different artefacts when publishing to nexus and locally which is hard for debugging.

I know that in order to publish all the sub projects you need to income tasks on all of them, we just don’t do it automatically.

The purpose of this change was to clarify usage of the release plugins and make the release and creation of Pom more transparent and consistent. Currently we have some weird Pom files preventing us to load h2o and sw artefacts in dbc and this was attempt to debug/fix it